recipes/elastic-apm/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "elastic-apm" %}
+{% set version = "6.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 7e5669556d9a228d5508c3f4925e3a7efd0cbf9203a4655e287408aa6bc53a84
+
+build:
+  number: 0
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
  number: 0
```